app.py

```
@@ -14,16 +14,25 @@
+                cur = mysql.connection.cursor()
+                cur.execute('''
+                CREATE TABLE IF NOT EXISTS messages (
+                    id INT AUTO_INCREMENT PRIMARY KEY,
+                    message TEXT
+                );
+                ''')
+                mysql.connection.commit()  
+                cur.close()
+            print("✅ Database initialized successfully.")
+            break
+        except MySQLdb.OperationalError as e:
```

Ruff (0.11.9)

Undefined name `MySQLdb`

(F821)